setup.py

```diff
@@ -17,6 +17,7 @@
     'imageio',
     'pillow',
     'appdirs',
+    'pooch',
```

Comment:
I am 50/50 on whether it would be good to have this as an `extras.require` instead.  In a significant fraction of use cases, users will never import `pyvista.examples`.  But, it makes getting started more frustrating.  Unfortunately, Python doesn't support a default extras for installation, which would solve this problem.

Thoughts?

Reply:
Pooch is lightwieght enough that I think it should be included by default - primarily for keeping the barrier to entry low so that users have tons of examples that will work out of the box

Reply:
+1 to @MatthewFlamm's comment about appdirs. I think it can now be safely removed